src/medals/constants.cljs

```diff
@@ -1,80 +1,79 @@
 (ns medals.constants)
 
 
-(def MEDAL_COLOR_CONSTANT [{:color_name "gold" :hex_code "#FFD700"} , {:color_name "silver" :hex_code "#D3D3D3"} ,{:color_name "bronze" :hex_code "#CD7F32"}])
+(def MEDAL_COLOR_CONSTANT [{:color_name :gold :hex_code "#FFD700"} , {:color_name :silver :hex_code "#D3D3D3"} ,{:color_name :bronze :hex_code "#CD7F32"}])
 
 (def ALLOWED_SORT_VALUES {:gold "gold"
                           :silver "silver"
                           :bronze "bronze"
                           :total "total"})
 
-
 (def ALLOWED_SUMMATION_VALUES {:silver  "silver"
                                :gold "gold"
                                :bronze "bronze"})
```

Comment:
You really only care about the keys here. Instead use a [set](https://clojure.org/reference/data_structures#Sets) data structure. They ensure there's no duplicates and you don't need the unnecessary values

Reply:
Acknowledged